modules/persistent-storage-csi-vsphere-install-issues.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+[id="persistent-storage-csi-vsphere-install-issues_{context}"]
+= Removing a non-Red Hat vSphere CSI Operator Driver
+
+If you have installed a non-Red Hat vSphere Container Storage Interface (CSI) Operator Driver, updates for your cluster to the next major version of {product-title}, such as 4.11, may be disabled due to {product-title} 4.10 having a built-in version of the vSphere CSI Operator Driver.
+
+{product-title} 4.10 clusters are still fully supported, and updates to bug fix versions of 4.10, such as 4.10.z, are not blocked, but you must correct this state by removing the non-Red Hat vSphere CSI Operator Driver before updates to next major version of {product-title} can occur. Removing the non-Red Hat vSphere CSI driver does not require deletion of associated persistent volume (PV) objects, and no data loss should occur.
+
+[NOTE]
+====
+These instructions may not be complete, so consult the vendor or community provider uninstall guide to ensure removal of the driver and components.
+====
+
+To uninstall the non-Red Hat vSphere CSI Operator Driver:
+
+. Delete the non-Red Hat vSphere CSI driver (VMware vSphere Container Storage Plug-in) Deployment and Daemonset objects.
+. Delete the configmap and secret objects that were installed previously with the non-Red Hat vSphere CSI Operator Driver.
+. Delete the non-Red Hat vSphereCSI driver `CSIDriver` object:
++
+[output, terminal]
+----
+~ $ oc delete CSIDriver csi.vsphere.vmware.com
+----
++
+[output, terminal]
+----
+csidriver.storage.k8s.io "csi.vsphere.vmware.com" deleted
+----
+
+If you have started an update to {product-title), after you have completed removal of non-Red Hat vSphere CSI driver, the {product-title} update automatically resumes.
```

Comment:
This statement is not accurate partially tbh. Presence of a non-Red Hat CSI driver never blocked update to 4.10 (unless you mean update to 4.11).

We should say something like:

Once you have removed non-Red Hat vSphere CSI driver from Openshift cluster, installation of Red Hat provided vSpehre CSI driver is automatically resumed and any conditions that could block upgrades to 4.11 are automatically removed. If you had existing vSphere CSI persistent-volume(PVs) objects, their lifecycle will be now managed by Red Hat provided vSphere CSI driver.

Reply:
Implemented this comment.

Reply:
The wording - " non-Red Hat vSphere CSI Operator Driver", should say "non-Redhat vSphere CSI driver". We can not assume the third party driver was installed via an Operator.

Reply:
Implemented this comment.